gradle.properties

```diff
@@ -49,3 +49,6 @@ android.defaults.buildfeatures.viewBinding=false
 
 # Suppress the warning about MPP being in alpha.
 kotlin.mpp.stability.nowarn=true
+
+# TODO: remove when updating to Kotlin 2.0
+kotlin.native.ignoreIncorrectDependencies=true
```

Comment:
Kotlin 1.x will warn on common compileOnly dependencies even if they are added to the non jvm source sets, 2.0 resolved that.